site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,305 @@
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/multiple_gamedirs.png"
+    title="Multiple Game Directories" >}}
+
+## [Add support for stereoscopic 3D](https://github.com/citra-emu/citra/pull/3632) by [N00byKing](https://github.com/N00byKing)
+
+If you are one of the lucky few with a 3D TV or monitor attached to your
+computer, stereoscopic 3D support has just been added to Citra! You can enable
+it by heading to <kbd><samp>Emulation</samp> &rarr; <samp>Configure...</samp> &rarr; <samp>Graphics</samp> &rarr; <samp>Layout</samp></kbd>,
+ticking the <samp>Enable Stereoscopic 3D</samp> checkbox, and changing the
+screen layout to <samp>Side by Side</samp>.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q2/3d-sxs.png"
+    title="Although you can't see this in 3D, try crossing your eyes or using a mirror!" >}}
+
+<!-- Holding off on this until tested
+ If you want to take a look at the results, I've also prepared a rendering in
+the 3DS' native 3D image format. Download this file and drop it into your SD
+card, or navigate to this link on your 3DS to take a look:
+[](/images/entry/citra-progress-report-2018-q2/CTRA0001.MPO) -->
+
+## [Software Keyboard Implementation](https://github.com/citra-emu/citra/pull/3850) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+For the longest time, many games were unplayable or needed workarounds on Citra
+due to them needing the software keyboard applet. The reason being that whenever
+they would try to call it, Citra didn't implement it, and so would simply tell
+the game that the user entered "Citra" and tapped OK, without actually prompting.
+But now, thanks to [zhaowenlan1779](https://github.com/zhaowenlan1779) and [jroweboy](https://github.com/jroweboy),
+it now has fully functional software keyboard emulation!
+
+Whenever a game requests it, Citra will pause the game and pull up a text box that
+you can fill in like any other prompt.
+
+{{< gifv src="/images/entry/citra-progress-report-2018-q2/swkbd.mp4"
+    title="Now you can change your name to anything!" >}}
+
+## Open Source System Archives ([#3977](https://github.com/citra-emu/citra/pull/3977), [#3881](https://github.com/citra-emu/citra/pull/3881)) by [B3n30](https://github.com/B3n30)
+
+Many Nintendo 3DS games require the use of system archives, that contain things
+like fonts, a bad word list, and assets for Miis. Because these are copyrighted,
+Citra could not include them, and users had to dump a copy from their consoles,
+which was a generally tedious and error-prone process.
+
+Now, [B3n30](https://github.com/B3n30) has created open source alternative fonts
+and a bad word list that can be used in games instead of the official archives.
+And, because these are open source, they are now included with Citra. If a game
+requires either archive, and you haven't dumped them yet, it'll automatically give
+the game the alternative archives, further decreasing Citra's dependency on
```

Comment:
`further decreasing Citra's dependency on copyrighted material.` could be taken the wrong way.